src/app/login/login.component.html

```diff
@@ -3,12 +3,22 @@
 <div class="main">
 	<div class="background">
 		<div class="loginbox">
-			<div class="titleBox">
-				<h3><span>Draw And Guess</span><span> Game</span></h3>
-			</div>
-				<input type="text" placeholder="username" class="user">
-				<input type="password" placeholder="password" class="pass">
-				<button>LOGIN</button>
+			<section>
+				<div class="titleBox">
+					<h3><span>Draw And Guess</span><span> Game</span></h3>
+				</div>
+				<div class="inputs">
+					<input type="text" placeholder="username" class="user">
+					<input type="password" placeholder="password" class="pass">
+					<input type="password" placeholder="re-password" class="re-pass" *ngIf="registration">
+					<button *ngIf="registration === false">LOGIN</button>
+					<button *ngIf="registration">REGISTRATION</button>
```

Comment:
better use [text-transform](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform) css property to make it uppercase